### DISCUSSION TITLE
faster way of importing xml schema files

### DISCUSSION THREAD

**Discussion by 10062244** [FreeOpcUa/opcua-asyncio#993] [Q&A] [answered]
Hello,
Is there anyway to more quickly import xml files? If I know the spec files I want to use, could i convert them to some other file type and then import? Or is there a way export my namespace to a bin file after importing xml's and start my server from that?

Thanks,

**Comment by schroeder-** [comment, marked as answer]
The performance of the import was improved in #963, maybe that helps.

**Reply by 10062244** [reply]
That made a huge difference, thanks!!!
